src/renderer/pages/project/script_editor/beat_editors/media.vue

```
@@ -131,60 +135,95 @@
+    }
+    return undefined;
+  })();
+  if (!imageType) {
+    notifyError(t("notify.error.media.unsupportedType", { fileType }));
+    return;
+  }
+
+  const extension =
+    imageType === "image"
+      ? fileType
+      : videoSubtypeToExtensions[fileType as keyof typeof videoSubtypeToExtensions];
```

ESLint

[error] Replace `⏎······?·fileType⏎·····` with `·?·fileType`

(prettier/prettier)